lntest/itest/lnd_test.go

```diff
@@ -10427,25 +10423,12 @@ func testAsyncPayments(net *lntest.NetworkHarness, t *harnessTest) {
 	// We should receive one insufficient capacity error, because we sent
 	// one more payment than we can actually handle with the current
 	// channel capacity.
-	errorReceived := false
 	for i := 0; i < numInvoices; i++ {
-		if resp, err := alicePayStream.Recv(); err != nil {
+		if _, err := alicePayStream.Recv(); err != nil {
```

Comment:
Is it worth replacing the `errorReceived` thing with a check for `TempChanFailure`?

Reply:
there's no way to reliably trigger it w/o holding the invoices on the receiver's side and filling the commitment, but if we're holding then it's not a good benchmark

Reply:
Ok, good point. In that case, the comment above is outdated.

Reply:
done